hardware/Schematic-Design-Guide.md

```diff
@@ -26,42 +33,58 @@ There are relative reference designs for all of the solutions in the website. Ba
+<div align="center">
+  <img src="files\HW-Schematic-Design-Guide\Proprietary-Reference.png">  
+</div>
+
+For example, If you want to look for 434MHz +10dBm output power reference design, you can click item "EFR32FG1 BRD4251B 2400/433 MHz 10dBm Radio Board Full Design Package" and download the reference design package to your local address.
+The schematic likes: [BRD4251B](https://www.Silabs.com/documents/public/schematic-files/EFR32FG1-BRD4251B-B00-schematic.pdf)
 
 But keep in mind that this reference design can be applied to a cluster of chips that have difference packages and frequency band configurations. So you can just replace the chips to your interest chip with different package and frequency band.
-## Hardware design considerations
-After you get the reference design, you may have the concern why the reference is designed like this. Therefore Silabs provides a lot of application notes or reference manual for customer reference. For basical hardware considerations, Customer can refer to the following application note for detail descriptions:[AN0002.1](https://www.silabs.com/documents/public/application-notes/an0002.1-efr32-efm32-series-1-hardware-design-considerations.pdf). And there are several hardware considerations in different divisions as following:
+# 3. Hardware design considerations
+After you get the reference design, you may have the concern why the reference is designed like this. Therefore Silicon Labs provides a lot of application notes or reference manual for customer reference. For basic hardware considerations, Customer can refer to the following application note for detail descriptions:[AN0002.1](https://www.Silabs.com/documents/public/application-notes/an0002.1-efr32-efm32-series-1-hardware-design-considerations.pdf). And there are several hardware considerations in different divisions as following:
 - Power supply configurations: 
   - General requirements: VREGVDD = AVDD, must be the highest voltage on  EFR32; DVDD,IOVDD,RFVDD,PAVDD =< AVDD; DECOUPLE < DVDD
   - If internal DC-DC is not used, typically tie all the power pins to the main power supply.
-  - If internal DC-DC is used, typically tie VREGVDD, AVDD, IOVDD to the main power supply, tie DVDD, RFVDD to DCDC output (VREGSW), tie PAVDD to DCDC output if TXP =< +13dBm, tie PAVDD to VMCU main supply if TXP > +13dBm.
+  - If internal DC-DC is used, typically tie VREGVDD, AVDD, IOVDD to the main power supply, tie DVDD, RFVDD to DCDC output (VREGSW), tie PAVDD to DCDC output if TXP(Transmit Power) =< +13dBm, tie PAVDD to VMCU main supply if TXP > +13dBm.
 
-  For more information, please refer to application notes: [AN0948](https://www.silabs.com/documents/public/application-notes/an0948-power-configurations-and-dcdc.pdf)
+  For more information, please refer to application notes: [AN0948](https://www.Silabs.com/documents/public/application-notes/an0948-power-configurations-and-dcdc.pdf)
```

Comment:
I think it is worthwhile here to point to AN0948 for series 1 and AN0948.2 for series 2.

Reply:
Added.